modules/cert-manager-configure-cloud-credentials-gcp-non-sts.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-prepare-cloud-credentials-gcp-non-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} on GCP
+
+You can configure cloud credentials for the {cert-manager-operator} on GCP.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a directory to store the `CredentialsRequest` custom resource (CR) of the {cert-manager-operator} by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Change the directory by running the following command:
++
+[source,terminal]
+----
+$ cd credentials-request
+----
```

Comment:
Ditto: https://github.com/openshift/openshift-docs/pull/59907/files#r1209215583
we can skip steps 1, 2 and directly move to what's in step 3